Jukebox/src/components/PlayList.vue

```diff
@@ -0,0 +1,25 @@
+<script setup>
+import { usePlayList } from "../composables/usePlayList.js";
+import Track from "./Track.vue";
+
+const {
+    tracks,
+    playedTrack,
+} = usePlayList();
+</script>
+
+<template>
+    <h2>Playlist</h2>
+    <ol v-for="track in tracks" :key="track.index">
+        <b v-if="track == playedTrack">
```

Comment:
Privilégiez la comparaison stricte. Ca s'applique sur tout votre code.
```suggestion
        <b v-if="track === playedTrack">
```